src/control/jobs/control_jobs.c

```diff
@@ -1850,29 +1852,35 @@ static int32_t dt_control_time_offset_job_run(dt_job_t *job)
            ngettext("adding time offset to %d image", "adding time offset to %d images", total), total);
```

Comment:
In both case the message displayed on-screen is "adding time offset", I think we should have another message when changing the date/time instead of the offset.

Reply:
Right!